funnel/templates/event.html.jinja2

```diff
@@ -24,7 +24,7 @@
       <form class="search-participant mui-form mui-form--margins">
         <div class="mui-textfield mui-textfield--float-label">
           <input autofocus class="field-title search-query" id="search" type="text" name="key" value=""/>
-          <label>Search</label>
+          <label><i class="material-icons mui--align-middle mui--text-title">search</i> Search</label>
```

Comment:
Doesn't this need ARIA labelling to prevent a TTS reader from saying "search" twice?